frappe_telegram/client.py

```diff
@@ -47,7 +47,7 @@ def send_message(message_text: str, parse_mode=None, user=None, telegram_user=No
     message = bot.send_message(telegram_user_id, text=message_text, parse_mode=parse_mode)
     log_outgoing_message(telegram_bot=from_bot, result=message)
 
-
+@frappe.whitelist()
```

Comment:
@Abadulrehman how did this creep in now ? 😅

Reply:
Oops. testing stuff. 😬